deployments/helm-chart/templates/controller-daemonset.yaml

```diff
@@ -80,6 +80,9 @@ spec:
           - -external-service={{ .Values.controller.reportIngressStatus.externalService }}
           - -enable-leader-election={{ .Values.controller.reportIngressStatus.enableLeaderElection }}
 {{- end }}
+{{- if .Values.controller.nginxStatus.allowCidrs }}
```

Comment:
please move it above, so it depends on   `{{- if .Values.controller.reportIngressStatus.enable }}`
no need for a separate condition for `{{- if .Values.controller.nginxStatus.allowCidrs }}`

Reply:
same for controller-deployment.yaml